### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by coderofsalvation**
+1 marketing! ;)

### CODE REVIEW

**Comment by polonskiy** [comment]
Do you using it on non x86/x86_64 architecture?

**Comment by coderofsalvation** [comment]
yes I do.
Btw. I also added hooks to it (see readme).
This allows me to have clean 'crowdr' repositories with only a crowdr.cfg.sh file and a hooks/ dir.
I store my dockerfiles in seperate repositories so I don't want to include them in my crowdr repos.

File: README.md [hunk 1]
```diff
@@ -159,6 +160,21 @@ container_name option value
 
 [Full reference](https://docs.docker.com/reference/commandline/cli/#run)
 
+### Hooks
+
+Every crowdr command can be extended.
+Lets say you want to pull in some Dockerfiles from remote repositories *before* running `crowdr build`.
+
+    $ mkdir hooks
+    $ echo 'pulling repos' > hooks/before.build
```
Comment (polonskiy) [inline README.md]: It will cause error.

File: README.md [hunk 2]
```diff
@@ -159,6 +160,21 @@ container_name option value
...
+    $ echo 'git clone http://github.com/someuser/docker.redis' > hooks/before.build
```
Comment (polonskiy) [inline README.md]: You are overwriting previous content.

File: README.md [hunk 3]
```diff
@@ -159,6 +160,21 @@ container_name option value
...
 
+### Hooks
+
+Every crowdr command can be extended.
+Lets say you want to pull in some Dockerfiles from remote repositories *before* running `crowdr build`.
+
+    $ mkdir hooks
+    $ echo 'pulling repos' > hooks/before.build
+    $ echo 'git clone http://github.com/someuser/docker.redis' > hooks/before.build 
+    $ echo 'git clone http://github.com/someuser/docker.proxy' >> hooks/before.build
```
Comment (polonskiy) [inline README.md]: What is the convention here: `cmd.before` or `before.cmd`?

File: crowdr [hunk 4]
```diff
@@ -127,4 +128,6 @@ parse_cfg() {
 cmd="${1:-run}"
 shift
 parse_cfg
+[[ -x $CROWDR_HOOKDIR/$cmd.before ]] && $CROWDR_HOOKDIR/$cmd.before
```
Comment (polonskiy) [inline crowdr:131]: Convention mismatch?

**Comment by polonskiy** [comment]
You have some errors, but your hook idea is **awesome**! I really like it.

**Comment by coderofsalvation** [comment]
nice :) ah yes, the docs and the implementation dont match lol :)
I'll correct that.

Btw. the hooks are pretty useful for me already.
They allow me to monkeypatch some things before building and after running them:

```
 $ crowdr build
 pulling latest dockerfiles from repositories
 (..)
 patching dockerfiles
 (..)
 $ crowdr
 lemon_redis_lb
 lemon_nodejs-ndg
 lemon_postfix
 lemon_hipache
 setup bidirectional connect lemon_nodejs => hipache
 172.17.0.38 hipache lemon_hipache
 waiting for nodejs-ndg to boot up
 copying ssh keys
```

**Comment by polonskiy** [comment]
Added in [0.6.0](https://github.com/polonskiy/crowdr/releases/tag/0.6.0). Thanks!

**Pull request closed without merging**